pkg/api/healthcheck.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	lambdaTypes "github.com/aws/aws-sdk-go-v2/service/lambda/types"
+
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	"github.com/common-fate/apikit/apio"
+	"github.com/common-fate/apikit/logger"
+	"github.com/common-fate/common-fate/pkg/cfaws"
+	"github.com/common-fate/common-fate/pkg/deploy"
+	"github.com/common-fate/common-fate/pkg/storage"
+)
+
+func (a *API) AdminRunHealthcheck(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
```

Comment:
The healtcheck is implemented as a service, so instead of invoking the healthcheck lambda function, it would be better here to add the hgeathcheck service to the API struct

then

```
err := a.Healthcheck.Check(ctx)
```

You should ensure that the rest API has IAM permission to invoke the Handlers. 
Be sure you test this in your live dev deployment as well.